tests/utils.go

```diff
@@ -1120,10 +1118,14 @@ func createSeparateDeviceHostPathPv(osName, nodeName string) {
 }
 
 func CreateHostPathPv(osName string, hostPath string) string {
-	return CreateHostPathPvWithSize(osName, hostPath, "1Gi")
+	return createHostPathPvWithSize(osName, hostPath, "1Gi")
 }
 
-func CreateHostPathPvWithSize(osName string, hostPath string, size string) string {
+func createHostPathPvWithSize(osName string, hostPath string, size string) string {
```

Comment:
In go if you have multiple parameters with a single type you can specify the type only one like this:
```go
func createHostPathPvWithSize(osName, hostPath, size string) string {
...
}
```